lib/node_modules/@stdlib/blas/ext/base/drssors/README.md

```diff
@@ -0,0 +1,339 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# drssors
+
+> Calculate the [residual sum of squares][wikipedia-residual-sum-of-squares] of two double-precision floating-point strided arrays, ignoring `NaN` values and using an ordinary recursive summation.
```

Comment:
@nakul-krishnakumar If this package is supposed to ignore `NaN` values, it is misnamed. It should be `dnanrssors`. For `drssors`, we shouldn't have any special NaN handling.

Accordingly, in order to avoid unnecessary work, I suggest simply renaming this package (and all aliases) from `drssors` to `dnanrssors`.